### DISCUSSION TITLE
Laravel 5.6 to Laravel 8 upgrade issue : Class config does not exist

### DISCUSSION THREAD

**Discussion by aherrao** [laravel/framework#41848] [Q&A] [answered]
- Laravel Version: 8.0
- PHP Version: 7.4

### Description:
We are using Laravel library in the core PHP project and wanted to update the major version to Laravel-8 but we are facing some internal issues when using helpers like url(), route(), etc.

Also, we don't have php artisan in the project so 

`php artisan config:clear ` can't use

### Steps To Reproduce:

Just upgrade to laravel-8 from 5.6

`**Fatal error**: Uncaught ReflectionException: Class config does not exist in /var/www/wmd/current/wmdcore-nightly/vendor/laravel/framework/src/Illuminate/Container/Container.php:877 Stack trace: #0 /var/www/wmd/current/wmdcore-nightly/vendor/laravel/framework/src/Illuminate/Container/Container.php(877): ReflectionClass->__construct('config') #1 /var/www/wmd/current/wmdcore-nightly/vendor/laravel/framework/src/Illuminate/Container/Container.php(758): Illuminate\Container\Container->build('config') #2 /var/www/wmd/current/wmdcore-nightly/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(851): Illuminate\Container\Container->resolve('config', Array, true) #3 /var/www/wmd/current/wmdcore-nightly/vendor/laravel/framework/src/Illuminate/Container/Container.php(694): Illuminate\Foundation\Application->resolve('config', Array) #4 /var/www/wmd/current/wmdcore-nightly/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(836): Illuminate\Container\Container->make('config', Array) #5 /var/www/wmd/current/w in /var/www/wmd/current/wmdcore-nightly/vendor/laravel/framework/src/Illuminate/Container/Container.php on line 879`

**Comment by aherrao** [comment, marked as answer]
Adding the following code to the file bootstrap.php solved my issue.

```
use Illuminate\Http\Request;
use Illuminate\Foundation\Http\Kernel;

$router = app('router');
$kernel = new Kernel($app, $router);
$kernel->handle(
    $request = Request::capture()
);
```

**Reply by tarekshaker1690** [reply]
In which file ?

**Reply by anegaprabhu** [reply]
In which file we have to add the above code ?

**Reply by aherrao** [reply]
bootstrap.php

**Reply by chancle** [reply]
I have the same issue, but I don't see the file you mentioned in the path.

**Reply by manikbd** [reply]
I am using laravel 10.  where is located bootstrap.php
